lib/electric_slide/call_queue.rb

```diff
@@ -216,9 +216,10 @@ def return_agent!(*args)
 
     # Removes an agent from the queue entirely
     # @param [Agent] agent The {Agent} to be removed from the queue
+    # @param [extra_params] hash Application specific extra params
```

Comment:
I think this documentation is correct. The expected class name of the argument should be inside the square brackets, and the name of the variable should be outside, eg:

```
# @param [Hash] extra_params ...
```